front/components/app/DatasetView.tsx

```diff
@@ -643,7 +650,7 @@ export default function DatasetView({
                           "font-mono w-full resize-none border-0 bg-transparent px-1 py-0 text-[13px] font-normal italic placeholder-gray-400 ring-0 focus:ring-0",
                           readOnly
                             ? "text-gray-500"
-                            : "text-gray-700 dark:text-gray-700-night"
+                            : "dark:text-gray-700-night text-gray-700"
```

Comment:
This and below are just from automatic linting, classes were not in the right order. 💅🏻